### ISSUE TITLE
It looks like you're using the development build of the Firebase JS SDK

### ISSUE THREAD

**Issue by avelow** [dappsnation/akita-ng-fire#33]
I am getting this in my console since i use akita-ng-fire.

```
It looks like you're using the development build of the Firebase JS SDK.
When deploying Firebase apps to production, it is advisable to only import
the individual SDK components you intend to use.

For the module builds, these are available in the following manner
(replace <PACKAGE> with the name of a component - i.e. auth, database, etc):

CommonJS Modules:
const firebase = require('firebase/app');
require('firebase/<PACKAGE>');

ES Modules:
import firebase from 'firebase/app';
import 'firebase/<PACKAGE>';

Typescript:
import * as firebase from 'firebase/app';
import 'firebase/<PACKAGE>';
```

I will try to do a pull request to fix this by changing the import of firebae in the library

**Comment by avelow** [comment]
https://github.com/dappsnation/akita-ng-fire/pull/34

**Comment by avelow** [comment]
should be fixed in https://github.com/dappsnation/akita-ng-fire/pull/34

**Comment by GrandSchtroumpf** [comment]
Should be fixed in version `akita-ng-fire@1.5.13`.
